ssl/ech/ech_internal.c

```diff
@@ -11,11 +11,78 @@
 #include <openssl/ech.h>
 #include "../ssl_local.h"
 #include "ech_local.h"
+#include <openssl/rand.h>
+#include "../statem/statem_local.h"
```

Comment:
Why is this include needed?

Reply:
for `ssl_cipher_list_to_bytes()`

Reply:
Hm... I'd prefer refactoring that function out into `ssl_ciph.c`. Otherwise this looks like a violation of encapsulation.